apps/req.c

```diff
@@ -313,6 +317,12 @@ int req_main(int argc, char **argv)
         case OPT_MULTIVALUE_RDN:
             multirdn = 1;
             break;
+        case OPT_EXTENSION:
+            if (extension_bio == NULL) {
+                extension_bio = BIO_new(BIO_s_mem());
```

Comment:
Would check for NULL here be appropriate? With fatal consequences. One can also argue that one should check for BIO_printf return value. Rationale would be that it's more appropriate to terminate than pass partial information (in case it succeeded at first option, but failed at second.